Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -297,7 +295,7 @@ open class _ComposerVC<ExtraData: ExtraDataTypes>: _ViewController,
 
         let isAttachmentButtonHidden = !content.isEmpty || channelConfig?.uploadsEnabled == false
         let isCommandsButtonHidden = !content.isEmpty || channelConfig?.commands.isEmpty == true
-        let isShrinkInputButtonHidden = content.isEmpty || (isAttachmentButtonHidden && isCommandsButtonHidden)
+        let isShrinkInputButtonHidden = content.isEmpty
```

Comment:
This will make shrink button appear when channel config doesn't allow uploads or there's no commands (or both), you need to test these cases too

Reply:
It's a temporary change I made in order multiple images/videos can be attached (will roll it back). But it seems the way we manage buttons requires one more iteration: ATM after adding an attachment neither 📎 nor shrink button are visible

Reply:
@evsaev @b-onc This was introduced by: c13e21abdfdb90431cfbc2889b87c92cc071a22d and I've opened a PR to fix it here: #1196